docs/intro.rst

```diff
@@ -53,6 +58,50 @@ Why yet another case-insensitive dictionary: We found that all previously
 existing case-insensitive dictionary packages on Pypi either had flaws, were
 not well maintained, or did not support the Python versions we needed.
 
+
+.. _`Overriding the casefold method`:
+
+Overriding the casefold method
```

Comment:
should it be Overriding the default casefold method

Reply:
Yes, DONE